src/merge-folders-spec.js

```diff
@@ -0,0 +1,183 @@
+const path = require('path')
+const assert = require('assert')
+
+const isFunction = x => typeof x === 'function'
```

Comment:
why taking `check-more-types` out just to write this little function then? Why not use `R.is` then?